src/components/protocol_handler/src/handshake_handler.cc

```diff
@@ -101,6 +101,7 @@ void HandshakeHandler::OnHandshakeFailed() {
   }
   ProcessFailedHandshake(params);
   bson_object_deinitialize(&params);
+  return true;
```

Comment:
@AKalinich-Luxoft Why do you need to return `true` always?

Reply:
@AByzhynar return result of this function is used by security manager to decide, should it delete this listener from the list or not. This is a reason why AM returns false in the same function.

Reply:
@AByzhynar if you look, the similar implementation in OnHandshakeDone function